src/views/account/InvitationCard.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+    <div>
+        <section class="modal-card-body">
+            <h3 class="title has-text-centered has-text-dark">Complete invitation card:</h3>
+            <div class="box">
+                <p>Enter the email of school headmaster:</p>
+                <b-field>
+                    <b-input v-model="applyingRequest.directorEmail" type="email" @change="emailCorrectlyFilled"
+                             placeholder="Email" required></b-input>
+                </b-field>
+                <p>Specify the position on which you are applying:</p>
+                <b-select v-model="applyingRequest.position" @change="onChange" expanded class="positionList">
+                    <option>Teacher</option>
```

Comment:
Can we extract this to some constant variable please? So we can reuse the position as enumerated list of some sort instead of hardcoded values